crates/rune/src/modules/char.rs

```diff
@@ -23,6 +24,10 @@ pub fn module() -> Result<Module, ContextError> {
     Ok(module)
 }
 
+fn char_to_int_impl(value: char) -> Result<Value, VmError> {
+    Ok((value as i64).into())
```

Comment:
Not sure this will behave as intended on high glyphs. Or maybe wrapping when the unsigned value should be intended (just to follow the bit pattern). I'll accept this for now, but we might want to consider something different in the future for integer conversion.